__tests__/UrqlClient_test.re

```diff
@@ -54,4 +54,17 @@ describe("UrqlClient", () => {
       Expect.(expect(client) |> toMatchSnapshot);
     });
   });
+
+  describe("Client with exchanges provided", () =>
```

Comment:
Just verifying that the `Client` can actually accept an exchanges array and pass the compiler.